configure.rs

```diff
@@ -0,0 +1,84 @@
+use std::env;
```

Comment:
```suggestion
//! Configuration that is shared between `compiler_builtins` and `testcrate`.

use std::env;
```

Reply:
Applied, thanks!

Reply:
Could you change to `//!`? Right now it is describing the `std::env` import :)

Reply:
Ah, I tried that, but it failed with:
```
error[E0753]: expected outer doc comment
 --> testcrate/../configure.rs:1:1
  |
1 | //! Configuration that is shared between `compiler_builtins` and `testcrate`.
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2 | use std::env;
  | ------------- the inner doc comment doesn't annotate this `use` import
  |
help: to annotate the `use` import, change the doc comment from inner to outer style
  |
1 | /// Configuration that is shared between `compiler_builtins` and `testcrate`.
  |   ~
For more information about this error, try `rustc --explain E0753`.
error: could not compile `testcrate` (build script) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

Reply:
Adding a new line doesn't help either, fwiw:
```
error[E0753]: expected outer doc comment
 --> testcrate/../configure.rs:1:1
  |
1 | //! Configuration that is shared between `compiler_builtins` and `testcrate`.
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2 |
3 | use std::env;
  | ------------- the inner doc comment doesn't annotate this `use` import
  |
help: to annotate the `use` import, change the doc comment from inner to outer style
  |
1 | /// Configuration that is shared between `compiler_builtins` and `testcrate`.
  |   ~
```

Reply:
Ah I guess that doesn't work well within `include!()`. We can call it good with a non-doc `//`

Reply:
OK, done. :+1: